G1ANT.Addon.PDF/PdfCompareCommand.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using G1ANT.Language;
+using BitMiracle.Docotic.Pdf;
+
+namespace G1ANT.Addon.PDF
+{
+    [Command(Name ="pdf.compare", Tooltip = "Check if two pdf files are the same")]
+    public class PdfCompareCommand : Command
+    {
+        public PdfCompareCommand(AbstractScripter scripter) : base(scripter)
+        {
+        }
+
+        public class Arguments : CommandArguments
+        {
+            [Argument(Name = "firstpdfpath", Required = true, Tooltip = "Path to the first pdf file")]
+            public TextStructure FirstPdfPath { get; set; }
```

Comment:
Can you make argument names shorter? eg. Path1, Path2?

Reply:
Yes